src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java

```diff
@@ -724,5 +724,69 @@ public void markRowAsInvalid() {
             this.validRow = false;
         }
     }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailNoClean() {
+        String bankAccountNumber = "12345";
+        validateCleanPayeeACHAccountExtractDetail(bankAccountNumber, bankAccountNumber);
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailCleanDashes() {
+        String bankAccountNumber = "1-2---3-4-5----6-";
+        String expectedBankAccount = "123456";
+        validateCleanPayeeACHAccountExtractDetail(bankAccountNumber, expectedBankAccount);
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailCleanDashesWithLetters() {
+        String bankAccountNumber = "A1-2---3-4-5---6-";
+        String expectedBankAccount = "A123456";
+        validateCleanPayeeACHAccountExtractDetail(bankAccountNumber, expectedBankAccount);
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithLetters() {
+        String bankAccountNumber = "A12345";
+        validateCleanPayeeACHAccountExtractDetail(bankAccountNumber, bankAccountNumber);
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithNull() {
+        validateCleanPayeeACHAccountExtractDetail(null, null);
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithEmptyString() {
+        validateCleanPayeeACHAccountExtractDetail(StringUtils.EMPTY, StringUtils.EMPTY);
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithSpaces() {
+        validateCleanPayeeACHAccountExtractDetail(" 1 23  4 5 6 ", "123456");
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithSpacesAndDashes() {
+        validateCleanPayeeACHAccountExtractDetail(" 1 2----3  4 - 5 6-7 ", "1234567");
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithSpacesAndDashesAndNumbers() {

```

Comment:
Should this unit test method be renamed from "...AndNumbers" to "...AndLetters", to more accurately reflect the non-numeric characters that are being tested by this test case?

Reply:
oh, yes, I will change